hardhat.config.js

```diff
@@ -1,36 +1,16 @@
 require("@nomicfoundation/hardhat-ethers");
 require('@nomicfoundation/hardhat-toolbox');
-require("dotenv").config();
+require("dotenv").config();  // Load environment variables
 
-const { ALCHEMY_RPC_URL, PRIVATE_KEY, ETHERSCAN_API_KEY } = process.env
+const { ALCHEMY_RPC_URL, PRIVATE_KEY } = process.env;
 
 module.exports = {
   solidity: "0.8.1",
   networks: {
-    base: {
-      url: process.env.ALCHEMY_RPC_URL,
```

Comment:
This is the correct way of passing the secret keys here from the .env file. Rather than passing them directly like this.  baseSepolia: {
      url: ALCHEMY_RPC_URL,  // Use Base Sepolia RPC URL from .env
      chainId: 534351,       // Base Sepolia chain ID
      accounts: [PRIVATE_KEY],  // Private key from .env
    }, This basically exposes all the keys